.gitignore

```diff
@@ -3,6 +3,7 @@ node_modules
 build
 dist
 .cache
+.idea
```

Comment:
Where's this from? Just curious

Reply:
It's something from my IDE (Webstorm), so I just added it to not accidentally commit it